### ISSUE TITLE
Error on: azd auth login. ERROR: logging in: listen tcp: lookup localhost on 8.8.8.8:53: no such host

### ISSUE THREAD

**Issue by Erik-Nordahl** [Azure-Samples/azure-search-openai-demo#143]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [X ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>I followed the guide and installed all prerequisites for local install. First use of azd: got above error.
I tried to uninstall WSL (Ubuntu) and Docker but no success. Also tried added '127.0.0.1 localhost' entry to HOSTS file.

### Any log messages given by the failure
>  typed: 'azd auth login', and got: ERROR: logging in: listen tcp: lookup localhost on 8.8.8.8:53: no such host

### Expected/desired behavior
> to be logged in.

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
Win 11.

### Versions
>  azd version 0.8.0-beta.2 (commit 429b6ccf9a9d1616efa8988f2f8561deeb464c71)

### Mention any other details that might be useful
I also installed Azure CLI (az) which works for login.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by garyjkuehn** [comment]
same issue here with beta.2.  beta.1 worked but when upgrade message appeared decided to upgrade. Azure CLI (az) works fine as well.  In debug mode the below is presented

PS C:\ADOopenai> azd auth login --debug
2023/04/25 14:35:08 main.go:169: using cached latest version: 0.8.0-beta.2 (expires on: 2023-04-26T18:05:02Z)
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-list-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-get-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-set-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-unset-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-reset-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-config-list-alpha-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-env-set-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-set-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-select-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-env-new-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-new-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-list-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-env-refresh-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-refresh-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-env-get-values-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-env-get-values-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-infra-create-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-infra-create-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-infra-delete-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-infra-delete-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-pipeline-config-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-pipeline-config-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-telemetry-upload-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-template-list-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-template-show-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-auth-token-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-auth-token-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-auth-login-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-auth-login-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-auth-logout-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-version-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-version-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-show-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-show-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-login-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-login-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-logout-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-init-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-init-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-restore-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-restore-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-build-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-build-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-provision-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-provision-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-package-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-package-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-deploy-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-deploy-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-up-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-up-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-monitor-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-monitor-action'
2023/04/25 14:35:08 cobra_builder.go:172: registering flags for action 'azd-down-action'
2023/04/25 14:35:08 cobra_builder.go:192: registering resolver for action 'azd-down-action'
2023/04/25 14:35:08 cobra_builder.go:133: Resolved action 'azd-auth-login-action'
2023/04/25 14:35:08 middleware.go:124: running middleware 'debug'
2023/04/25 14:35:08 middleware.go:124: running middleware 'telemetry'

ERROR: logging in: listen tcp: lookup localhost on 192.168.0.1:53: no such host

**Comment by vhvb1989** [comment]
This is a regression on azd latest version.
It is already fixed on `main` branch with: https://github.com/Azure/azure-dev/pull/2010

If you are blocked by this, you can install a patch azd version from the PR following the instructions from here: https://github.com/Azure/azure-dev/pull/2010#issuecomment-1520521095

**Comment by vhvb1989** [comment]
Fixed on azd version >= 0.9.0
